src/app/analyze/stack/container-toggler/container-toggler.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, ViewEncapsulation, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'container-toggler',
```

Comment:
The selector needs to have a unique prefix to namespace it. For example we are already have 'f8-' and 'alm-'. You may use 'f8-' or come up with your own.